src/sentry/testutils/cases.py

```diff
@@ -43,6 +43,7 @@
 from django.test.utils import CaptureQueriesContext
 from django.utils import timezone
 from exam import before, fixture, Exam
+from multiprocessing.dummy import Pool as ThreadPool
```

Comment:
Since we're allowed to use modules beyond what's provided in the stdlib here, can you please change to `ThreadPoolExecutor`? This was discussed in https://github.com/getsentry/sentry/pull/19014#discussion_r430563043.

Reply:
Dang, thanks for the heads up! Doing now.